packages/apps-config/src/api/spec/encointer-node.ts

```diff
@@ -22,8 +22,14 @@ export default {
   AttestationIndexType: 'u64',
   CurrencyIdentifier: 'Hash',
```

Comment:
this should actually be represented in base58 encoding, not as hex. but I don't know (yet) how to do that.